### PR TITLE
Adds optional hebrew layout (Unicode)

### PR DESCRIPTION
**Pull request by josjoha**
## Description

This is a dual layout, for which two layers get switched when a layout is switched. There already is a bunch of layout options, and this adds a hebrew option. The hebrew is defined in Unicode. The files added for this layer are a C code file, a manual, a header for the Unicode definitions, and a header file for the options. The hebrew comes in three variations, to work well with Qwerty, Dvorak, or to be an Israeli layout.

In addition to this, the led on/off buttons now react immediately when pressed (simple one line change).
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* Not being able to type hebrew natively on the keyboard.

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — APPROVED** [review]

File: keyboards/thevankeyboards/minivan/keymaps/josjoha/unicode_hebrew.h [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+ * Author: © 2021 by Jos Boersema
+ *
+ */
```
Comment (fauxpark) [inline keyboards/thevankeyboards/minivan/keymaps/josjoha/unicode_hebrew.h:19]:
```suggestion
 */
 
 #pragma once
```

File: keyboards/thevankeyboards/minivan/keymaps/josjoha/minifan_config_compact.h [hunk 2]
```diff
@@ -14,7 +14,7 @@ GNU General Public License for more details.
 You should have received a copy of the GNU General Public License
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
- * © 2020,2021 by Jos Boersema
+* © 2020,2021 by Jos Boersema
  */
```
Comment (fauxpark) [inline keyboards/thevankeyboards/minivan/keymaps/josjoha/minifan_config_compact.h:18]:
```suggestion
 */
 
#pragma once
```

File: keyboards/thevankeyboards/minivan/keymaps/josjoha/base_hebrew.h [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+ * Author: © 2020 by Jos Boersema
+ *
+ */
```
Comment (fauxpark) [inline keyboards/thevankeyboards/minivan/keymaps/josjoha/base_hebrew.h:19]:
```suggestion
 */

#pragma once
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Comment by josjoha** [comment]
Hello fauxpark / others. Could you explain why this is necessary ? Why these files and not others, such as base_numpad.h ? I don't see a need for this to be honest. They won't be included more than once, it should not be possible, and if it happens it should be a programming mistake which should and I would believe can be fixed. If it is required that I do this, of course I'm happy to add it, but it seems to be chaotic to do it for some files and not others, and I see no need, therefore please advice, thanks.

**Comment by fauxpark** [comment]
> They won't be included more than once, it should not be possible, and if it happens it should be a programming mistake which should and I would believe can be fixed.

That may be, but adding include guards (or in our case `#pragma once`) to header files is a good habit to get into, just in case.

> Why these files and not others, such as base_numpad.h ?

You didn't change that file in this PR, so I can't add a suggestion to it. I hoped you would get the idea.

**Comment by josjoha** [comment]
I see. Well, at least in this case, I personally do not agree with it because I think it is a good idea to see errors when there is messy code which ends up including header files more than once, but since pull requests are supposed to follow the QMK conventions, I'll add “#pragma once” to all such files. Thanks for your input (will do it later, no time right now). cheers

**Review by fauxpark — APPROVED** [review]

**Pull request merged**